updatehub/tests/common.rs

```diff
@@ -39,9 +39,11 @@ pub struct Settings {
 
 impl Default for Settings {
     fn default() -> Self {
+        let listen_socket = listen_available_port().unwrap().to_string();
```

Comment:
Please use an `.expect(...)` so we have a good panic information if it ever happens. Another thing is if we need to use `.to_string()` here as we can use it as a number on the `format!` call, bellow.